proto/v1/cdk.proto

```diff
@@ -52,3 +60,22 @@ message HoneyventRequest {
   int64 duration_ms = 4;
 }
 
+message WriteCacheRequest {
+  string key = 1;
+  bytes data = 2;
+  google.protobuf.Timestamp expires = 3;
+}
+
+message ReadCacheRequest {
+  string key = 1;
+}
+
+message ReadCacheResponse {
+  bool miss = 1;
+  optional bytes data = 2; // body is empty if cache miss
+}
+
+message WriteCacheResult {
+  bool success = 1;
+  optional string msg = 2; // error message is empty if success true
```

Comment:
nit: In Go, we are used to do `if err != nil` so, could we flip this to be `error` so we can do:
* `if response.Error` instead of
* `if !response.Success`
```suggestion
  bool error = 1;
  optional string message = 2; // error message is empty if error false
```

Reply:
Will revise.

Reply:
Fixed in cf85c3009a3e40c0df3c2d526721c3f99b8a56be